apps.json

```diff
@@ -1,4 +1,12 @@
 [
+{
+  "name": "NUS Buses",
+  "description": "Download the NUS Buses app to get real up to date bus arrival timings, bus routes and easily favourite the bus stops you prefer. It comes with a modern UI that is responsive and intuitive. Still in active development(2018)",
+  "author": "Marcus Koh Chi Hao",
+  "url": "https://goo.gl/jSDnQM",
+  "icon_url": "https://scontent.fsin3-1.fna.fbcdn.net/v/t1.0-1/p480x480/26113864_141658423281922_7246164886583028167_n.png?oh=54f8df88a59e21963907eb30d2d1319e&oe=5AFCD837",
+  "tags": ["nus", "buses"]
```

Comment:
Probably shouldn't have an "nus" tag since this is an NUS-only site after all haha